lib/kcerts/ssh.go

```diff
@@ -141,11 +155,18 @@ func (a *SSHAgent) UseStandardPaths() error {
 	tempname := fmt.Sprintf("%s/enkit.tmp%016x", path, mathrand.New(srand.Source).Uint64())
 	defer os.Remove(tempname)
 	if err := os.Symlink(a.Socket, tempname); err != nil {
-		return fmt.Errorf("UseStandardPaths symlink failed: %w", err)
+		return AnnotatedErrorf("UseStandardPaths symlink failed: %w", err)
+	}
+	if _, err := os.Stat(standard_socket_path); err == nil {
+		// The file exists, so let's remove it or die trying.
+		err = os.Remove(standard_socket_path)
```

Comment:
the remove should not be necessary. Before adding more code to enkit, can you write a small test showing that remove is necessary?

Also, with your last changes the message is certainly uniquely identifiable. Don't disagree it's a nice to have feature, would move under lib/... not sure, utils?